frontend/www/js/omegaup/components/group/Identitiesv2.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="card">
+    <div class="card-body">
+      <div class="upload-csv mb-4">
+        <div>
+          <omegaup-markdown :markdown="T.groupsCsvHelp"></omegaup-markdown>
+          {{ T.groupsUploadCsvFile }}
+          <input
+            name="identities"
+            type="file"
+            accept=".csv,.txt"
+            @change="readCsv"
+          />
+        </div>
+      </div>
+      <div v-show="identities.length > 0" class="card no-bottom-margin">
```

Comment:
```suggestion
      <template v-show="identities.length > 0">
```

como que tener cards anidadas no es tan buena idea.

Reply:
Listo